Front-End/Javascript/Javascript-Basics/mathOperator.js

```diff
@@ -0,0 +1,8 @@
+let number1 = 100;
+let number2 = 200;
+
+console.log("Addition is : ",number1 + number2);
```

Comment:
These all are the simple mathematical operations on the given values in Javascript